packages/core/src/utils/styleUtils/stylesUtils.ts

```diff
@@ -98,7 +107,26 @@ export const buildCfStyles = ({
     objectFit: cfImageOptions?.objectFit,
     objectPosition: cfImageOptions?.objectPosition,
   };
+  const filteredRules = Object.fromEntries(
```

Comment:
I think before it was called `cssWithoutUndefined`. I am still a fan of that name vs `filteredRules`. It tells me exactly what that variable holds.